imap_processing/cli.py

```diff
@@ -353,7 +354,7 @@ def pre_processing(self):
         return self.download_dependencies()
 
     @abstractmethod
-    def do_processing(self, dependencies: list):
+    def do_processing(self, dependencies: list):  # type: ignore[no-untyped-def]
```

Comment:
What does "ignore[no-untyped-def]" mean?

Reply:
It is a mypy inline ignore error command that skips over the error raised. In this case, it is needed because the function itself is not implemented. Once the function is implemented the command can be removed and a return type would need to be specified. I will add a comment noting that the command will need to be removed later.